DEVELOPER_GUIDE.md

```diff
@@ -54,7 +54,6 @@ nvm use --install
 
```

Comment:
Revert this change

Reply:
complete by rewriting the changes to this branch (with the changes from main & the edit to the button needed to close this ticket).